src/locales/en.json

```diff
@@ -302,23 +307,37 @@
     "TOTAL": "Total",
     "AMOUNT_PLUS_FEES": "Amount + fees",
     "REVIEW_TRANSACTION": "Review transaction",
+    "SIGN_TRANSACTIONS": "Sign {{count}} transactions",
     "AMOUNT": "Amount",
     "YOUR_ADDRESS": "Your address",
     "FROM": "From",
     "INPUT": "Inputs",
     "INPUT_AND_OUTPUT": "Inputs & Outputs",
-    "OUTPUT": "Output",
+    "OUTPUT": "Outputs",
     "RECIPIENT": "Recipient",
     "ASSET": "Asset",
+    "CLOSE": "Close",
+    "SIGNING_TRANSACTIONS": "Signing transactions",
+    "BROADCASTING_TRANSACTIONS": "Broadcasting transactions",
+    "THIS_MAY_TAKE_A_FEW_MINUTES": "This may take a few minutes, please keep this window open.",
+    "TRANSACTIONS_BROADCASTED": "Transactions broadcasted",
+    "TRANSACTIONS_FAILED_TO_BROADCAST": "{{failed}}/{{total}} transactions failed to broadcast",
+    "CHECK_THE_APP_YOU_USE": "Please check the app you are using.",
+    "TRANSACTIONS_SIGNED": "Transactions signed",
     "NETWORK_MISMATCH": "There’s a mismatch between your active network and the network you’re logged with.",
     "ADDRESS_MISMATCH": "There’s a mismatch between your signing address and the address you’re logged with.",
     "PSBT_NO_BROADCAST_DISCLAIMER": "This transaction will not be broadcasted from your wallet. It may be broadcasted later by a third party.",
+    "PSBTS_NO_BROADCAST_DISCLAIMER": "This transactions will not be broadcasted from your wallet. They may be broadcasted later by a third party.",
```

Comment:
Fix typo:
These transactions will not be broadcasted from your wallet. They may be broadcasted later by a third party.